vector/v.db.select/main.c

```diff
@@ -161,6 +186,31 @@ int main(int argc, char **argv)
                           options.file->answer);
     }
 
+    plain = FALSE;
+    csv = FALSE;
+    json = FALSE;
+    vertical = FALSE;
+    if (strcmp(options.format->answer, "csv") == 0)
+        csv = TRUE;
+    else if (strcmp(options.format->answer, "json") == 0)
+        json = TRUE;
+    else if (strcmp(options.format->answer, "vertical") == 0)
+        vertical = TRUE;
+    else
+        plain = TRUE;
+    if (json) {
+        fatal_error_option_value_excludes_flag(options.format, flags.escape,
+                                               _("Escaping is based on the format"));
+        fatal_error_option_value_excludes_flag(options.format, flags.colnames,
+                                               _("Column names are always included"));
+        fatal_error_option_value_excludes_option(options.format, options.fsep,
+                                                 _("Separator is based on the format"));
+    }
+    if (!vertical) {
+        fatal_error_option_value_excludes_option(options.format, options.vsep,
+                                                 _("Only vertical output can use vertical separator"));
+    }
+
```

Comment:
```c
if (csv)
    fatal_error_option_value_excludes_option(options.format, options.fsep, _("Separator is based on the format"));
```
CSV (comma-separated values) is only comma-separated. Right now, fsep is configurable with comma as default for CSV, but can we call it CSV with a non-comma separator is used?

Reply:
Although comma is often expected for CSV, CSV is also used to refer to broader format group with same or roughly same syntax, but different separator as demonstrated by the csv package in Python. In some natural languages, comma is used as a decimal point which leads to semicolon being the most common separator (as comma may lead to confusion even when period is used as decimal point). This is visible even in some [GRASS helper code](https://github.com/OSGeo/grass/blob/9dd029564047cbd531431055cf244fd3a99aa185/man/parser_standard_options.py#L118). So, yes, we can call it CSV even if delimiter/separator is customizable.

Practically, allowing different separators gives users the choice to produce some flavors of TSV or DSV although the plain format may be more suitable for that.